src/main/java/uk/gov/companieshouse/confirmationstatementapi/service/PscService.java

```diff
@@ -32,4 +41,11 @@ public PscsApi getPscs(String companyNumber) throws ServiceException {
             throw new ServiceException("Error Retrieving Persons of significant control", e);
         }
     }
+
+    public List<PersonOfSignificantControlJson> getPSCsFromOracle(String companyNumber) throws ServiceException {
+        List<PersonOfSignificantControl> pSCs = oracleQueryClient.getPersonsOfSignificantControl(companyNumber);
```

Comment:
This need to be camel case?

Reply:
i thought it was PSCs = pSCs. intellij complains of spelling if i use pscs.

Reply:
updated, i can live with intellij's spelling warning